staking_deposit/settings.py

```diff
@@ -1,5 +1,5 @@
 from typing import Dict, NamedTuple
-from .version import __version__
```

Comment:
Ok I am not sure I am seeing the right commits here but this delete also needs to be undone

Reply:
Please rever this change to that on line 4` __version__` is available.

Reply:
Done, thanks!